tests/rptest/remote_scripts/compute_storage.py

```diff
@@ -9,21 +9,31 @@
 import json
 
 
+def safe_listdir(p: Path) -> list[Path]:
+    """
+    It's valid for directories to be deleted at any time, 
+    in that case that the directory is missing, just return
+    that there are no files.
+    """
+    try:
+        return list(p.iterdir())
+    except FileNotFoundError:
+        return []
```

Comment:
Can it be kept lazy with something like:
```suggestion
        return p.iterdir()
    except FileNotFoundError:
        return iter([])
```

Reply:
Thanks good suggestion, I ended up just fixing the return type